CHANGELOG.md

```diff
@@ -9,13 +9,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+### Fixed
+
+## [3.4.1] 2019-02-12
+*Released to Fire TV 4K.*
+### Added
+
+### Changed
+
 ### Fixed
 - Bug where cursor would disappear if on overlay during page load finish (#1732)
 - Bug that could cause a crash on startup (#1778)
 - Fixed links on settings About page that did not load (#1731)
 - A memory leak (#1628)
 - Fixed bug that could cause YouTube to display vertically offset from where it should be (#1719)
 
+## [3.4] - 2019-02-12
+*Released to all devices except Fire TV 4K.*
```

Comment:
Why didn't we release 3.4 to 4k? Does that mean they were left on 3.3.1 (which has the same functionality)?

It might help to include these details in the changelog so others don't have the same questions.

Reply:
No see the changelog change above - 4k is upgraded to 3.4.1